python/ray/train/data_parallel_trainer.py

```diff
@@ -373,6 +375,97 @@ def get_dataset_config(self) -> Dict[str, DatasetConfig]:
         """
         return self._dataset_config.copy()
 
+    def _ipython_display_(self):
+        try:
+            from ipywidgets import HTML, VBox, Tab, Layout
+        except ImportError:
+            logger.warn(
+                "'ipywidgets' isn't installed. Run `pip install ipywidgets` to "
+                "enable notebook widgets."
+            )
+            return None
+
+        from IPython.display import display
+
+        title = HTML(f"<h2>{self.__class__.__name__}</h2>")
+
+        tab = Tab()
+        children = []
+
+        tab.set_title(0, "Train Loop Config")
+        children.append(
+            self._train_loop_config_repr_html_() if self._train_loop_config else None
+        )
+
+        tab.set_title(1, "Backend Config")
+        children.append(
+            self._backend_config._repr_html_() if self._backend_config else None
+        )
+
+        tab.set_title(2, "Dataset Config")
+        children.append(
+            self._dataset_config_repr_html_() if self._dataset_config else None
+        )
+
+        tab.set_title(3, "Scaling Config")
+        children.append(
+            self.scaling_config._repr_html_() if self.scaling_config else None
+        )
+
+        tab.set_title(4, "Run Config")
+        children.append(self.run_config._repr_html_() if self.run_config else None)
+
+        tab.set_title(5, "Datasets")
+        children.append(self._datasets_repr_html_() if self.datasets else None)
```

Comment:
What do you think about reorganizing the tabs as "Datasets", "Dataset Config", "Train Loop Config", "Scaling Config", "Run Config", "Backend Config"? The rationale is the sort by application flow (Data -> Train) and then importance (e.g., `BackendConfig` isn't that important)

Reply:
Hey, that sounds reasonable to me :+1: